src/app/custom/translations.tsv

```diff
@@ -1064,3 +1064,6 @@
 "root_panel_link"	"Instances management"	"Gestion des instances"
 "admin_panel_link"	"Instance administration"	"Administration de l'instance"
 "subresource_path_validation_error"	"A subresource with this path already exists"	"Une sous-ressource avec ce chemin existe déjà"
+"import"	"Import"	"Importer"
+"import_successfull"	"Import completed successfully"	"Import réalisé avec succés"
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏

```suggestion
"import_successful"	"Import completed successfully"	"Import réalisé avec succès"
```